src/plugins/navigation/schemas/schema.graphql

```diff
@@ -345,6 +360,18 @@ input NavigationTreeItemInput {
   navigationItemId: ID!
 }
 
+"Input for the `createNavigationTree` mutation"
+input CreateNavigationTreeInput {
+  "An optional string identifying the mutation call, which will be returned in the response payload"
+  clientMutationId: String
+
+  "The field updates to apply"
+  navigationTree: NavigationTreeInput!
```

Comment:
You can't reuse `NavigationTreeInput` because we need `name` to be required for an insert. You can put `name` and `draftItems` directly in this input as I did in https://github.com/reactioncommerce/reaction/issues/6011#issuecomment-572713305